Source/WebCore/rendering/HitTestLocation.cpp

```diff
@@ -28,7 +28,7 @@ HitTestLocation::HitTestLocation() = default;
 
 HitTestLocation::HitTestLocation(const LayoutPoint& point)
     : m_point(point)
-    , m_boundingBox(LayoutRect { flooredIntPoint(point), LayoutSize { 1, 1 } })
+    , m_boundingBox(LayoutRect { point, LayoutSize { 1, 1 } })
```

Comment:
This is going to produce some interesting results at top/right/bottom/leftPadding() functions where we use rounded point. is that expected? also this class is full of explicit integral snapping calls. While this is good direction, it may break some other content.

Reply:
Yes - hence, it is draft and I want to see what breaks. :-)